_includes/markdown/Vue.md

```diff
@@ -0,0 +1,76 @@
+The purpose of this document is to help guide you through working with a potential project using the JavaScript framework, Vue. Outside of The [Official Vue Documentation](https://vuejs.org/), you should find here personalized recommendations for using the framework based on the types of projects we typically see at 10up. We have broken these recommendations out into common elements we tend to interact with often in client work. If something you’re looking for isn’t represented you can either submit a pull request to this repository or refer to the [Official Vue Documentation](https://vuejs.org/v2/guide/).
+
+For Vue-specific coding style and pattern recommendations please see the official [Vue style guide](https://vuejs.org/v2/style-guide/).
+
+## Figuring Out If Vue Is Right For Your Project
+
+Using the right tool for the job is critical in navigating a successful project. Will Vue always be the answer? Of course not. But there are some instances where you may want to use this framework over something like (for example) a collection of plugins, custom JavaScript, or another framework. For instance, if you're building a larger-scale application that you feel needs a stateful system, we've found that [React]({{ site.baseurl }}/react) is a often a better option when compared to Vue.
+
+Some questions you may want to ask yourself:
+* Will this project contain a lot of components that need to interact with an API?
+* Are you replicating a lot features already present in Vue like client side templating and state management?
+* Will the addition of a framework save time and performance versus creating a custom solution? 
+* Do your client side templates need to be accessible to a back end technology (like PHP in a WordPress template)?
+
+If you find yourself answering, “Yes,” to those questions, Vue may be a great fit for your project or component! If not, a more traditional build may be the best.
+
+If you're answering "no" to some of those questions, but still feel a JavaScript-framework solution is the correct model, you may want to read through our [React]({{ site.baseurl }}/react) recommendations as a path forward for a larger scale application. 
+
+## Implementing Vue on a Project
```

Comment:
After this page is published I'd like to revisit and some specifics of implementing Vue in a WP project. 

As mentioned, I feel like Vue is an excellent tool for medium-large sized features within a site, but using Vue within a WordPress environment has some nuance. 

Examples: 
What is a "medium-large" feature — we could include a few examples of search/filtering interfaces 
How do we recommend handling api queries?  
How do we handle sharing template-part partials between WordPress and Vue? 
etc.

I haven't read much out there about good ways of thinking about these WordPress + Vue problems, so I think our voice here would be a welcome one.  

Again, I wouldn't want this to hang up the PR process here, but rather supplement the content when this is published.

Reply:
@saltcod I'll flag this for discussion after initial merge.